README.md

```diff
@@ -63,5 +63,6 @@ The following outlines our release process:
 2. Bump package version in `packages/gestalt/package.json` & update `CHANGELOG.md`.
 3. Open a pull request with the new version and land that in master.
 4. Once the version is bumped in master, checkout that commit locally.
-5. Run the release script from the root directory of the project `./scripts/publish.js` to publish the tag, npm package, and docs.
-6. Draft a new release from the tag at https://github.com/pinterest/gestalt/releases.
+5. Run `npm login` using your npm username and password.
```

Comment:
@chrislloyd I'm ok with this for now, `yarn login` doesn't seem to work for us since it doesn't create a token to use with TTY